fe/fe-core/src/main/java/com/starrocks/catalog/MaterializedView.java

```diff
@@ -466,9 +470,16 @@ public String toString() {
     @SerializedName(value = "maxMVRewriteStaleness")
     private int maxMVRewriteStaleness = 0;
 
+    // Multi ref base table partition expression and ref slot ref map.
     @SerializedName(value = "partitionExprMaps")
     private Map<ExpressionSerializedObject, ExpressionSerializedObject> serializedPartitionExprMaps;
     private Map<Expr, SlotRef> partitionExprMaps;
+    // ref base table to partition expression
+    private Optional<Map<Table, Expr>> refBaseTablePartitionExprsOpt = Optional.empty();
```

Comment:
it seems that Optional should be used as type of parameter passing in/out functions. 
class field definition should not adopt Optional type.

use transient keyword before cached fields.